Package.swift

```diff
@@ -22,7 +22,7 @@ let package = Package(
             targets: ["PurchasesHybridCommonUI"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/RevenueCat/purchases-ios-spm", exact: "5.20.0"),
+        .package(url: "https://github.com/RevenueCat/purchases-ios-spm", exact: "5.20.2-vc-beta.1"),
```

Comment:
Not a huge deal but personally I prefer to keep version bumps off of the beta branches to avoid conflicts later. This should ideally just be bumped by the automations/fastlane commands when making a release. NABD though, FFTI!

Reply:
Yeah running the fastlane command with the beta version should bump the dependency